engine/apps/alerts/incident_appearance/templaters/web_templater.py

```diff
@@ -15,7 +15,7 @@ def _postformat(self, templated_alert):
         if templated_alert.title:
             templated_alert.title = escape_html(self._slack_format_for_web(templated_alert.title))
         if templated_alert.message:
-            message = escape_html(self._slack_format_for_web(templated_alert.message))
```

Comment:
Can we just enable HTML links here? (not escaping HTML coming from an alert payload and rendering it could lead to other issues, I guess) I think part of the issue may be the [slack formatter](https://github.com/grafana/oncall/blob/dev/engine/apps/slack/slack_formatter.py#L8) changing links?

Reply:
Thanks! that is probably the correct solution, I'll test and then revert or update things. I'm not sure why the slack formatting is part of the web formatting, it would be nice to separate if slack needs it differently.